react/AutocompleteResults.tsx

```
@@ -176,7 +219,7 @@ const AutocompleteResults = ({
                   // eslint-disable-next-line jsx-a11y/anchor-is-valid
                   <a
                     href="#"
-                    onClick={event => event.preventDefault()}
+                    onClick={(event) => event.preventDefault()}
```

for whoever sees this, ignore this error